source/manage/mmctl-command-line-tool.rst

```diff
@@ -7,15 +7,56 @@ Being installed locally enables System Admins for both self-managed and Cloud Ma
 
 This feature was developed to a large extent by community contributions and we'd like to extend our gratitude to the contributors who have worked on this project. We are currently accepting pull requests for Help Wanted issues in the `mattermost-server <https://github.com/mattermost/mattermost-server/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22+label%3AArea%2Fmmctl>`__ repo. You can learn more about the unit test coverage campaign for mmctl in the `Unit testing mmctl commands <https://mattermost.com/blog/unit-testing-mmctl-commands/>`__ blog post.
 
-**Notes**
+What's new in Mattermost v6.0?
+------------------------------
+
+The following new mmctl functionality is available from Mattermost v6.0:
+
+- Output streaming that includes a progress indicator, easy navigation via paged output, output to file support, and basic piped output search support.
+- It's easy to reporting mmctl issues to Mattermost using the link provided when unhandled errors and exceptions are encountered.
+- Interactive user confirmations.
+
+For more information on what's new in mmctl for Mattermost v6.0, see the `Mattermost v6.0 mmctl Demo video <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_.
```

Comment:
@justinegeffen - the two video links will be changing since they are currently placeholders only. I'm in the process of confirming the final URLs for the two YouTube videos with Marketing.